integration-cli/cli/cli.go

```diff
@@ -33,6 +33,12 @@ func DockerCmd(t testingT, args ...string) *icmd.Result {
 	return Docker(Args(args...)).Assert(t, icmd.Success)
 }
 
+// DockerCmd executes the specified docker command but not expect a success,
+// the caller should check the return value
+func DockerCmdNoAssert(args ...string) *icmd.Result {
+	return Docker(Args(args...))
```

Comment:
I don't think this helper is necessary. Just call `Docker(Args(...))` directly.

`DockerCmd()` only exists for backwards compatibility.

Reply:
We can upgrade the frozen image to the latest version and only support the latest for this test case 🉑 things will be easy as it should to be...